.github/workflows/publish.yml

```diff
@@ -113,3 +113,39 @@ jobs:
         run: echo '//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}' > .npmrc
       - run: npm publish
         working-directory: "oso-js-node-${{ github.event.inputs.version }}"
+  rust:
+    name: Publish Rust crates to Crates.io
+    runs-on: ubuntu-18.04
+    env:
+      CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-release-${{ hashFiles('**/Cargo.lock') }}
+      - name: Install Rust stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - name: package polar-core
+        working-directory: polar-core
+        run: |
+          cargo publish
+      - name: package polar-core
```

Comment:
polar-core is here twice

polar-capo and polar-wasm-api instead?

Reply:
Oh oops, and I dunno if we want capi or wasm since we don't publish those as crates.

Reply:
Oh probably not then.